modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -245,6 +247,60 @@ The other two commands are:
 
 TIP: See xref:configuration/server/background_jobs_configuration.adoc[background jobs configuration] to learn more.
 
+[[managing-background-jobs]]
```

Comment:
There's no need for creating an anchor manually, as it's exactly the same as what is auto-generated.